### DISCUSSION TITLE
为什么执着于react呢？

### DISCUSSION THREAD

**Discussion by ninja-liu** [ant-design/ant-design#40182] [Q&A]
只是个人的一个疑问，为什么执着于react呢？
现在大部分库要么基于原生、要么兼容很多框架，ant系列为什么要执着于react呢？

**Comment by ninja-liu** [comment]
另外，虽然冒昧。但是，若bing 国际版查recommended react UI framework，极少提到antdesign，如何得出第二流行的react ui库呢？

**Reply by yoyo837** [reply]
国内的搜索都不在bing

**Comment by li-jia-nan** [comment]
antd 也有 vue 版本和 ng 版本的，只是由社区在维护而已

**Comment by afc163** [comment]
能力和业务场景不足，维护不了多个版本的 antd。
